lib/mix/tasks/recode.ex

```diff
@@ -10,6 +10,8 @@ defmodule Mix.Tasks.Recode do
 
   Without a `wildcard` argument the `inputs` value from the config is used.
 
+  If `inputs` value is `-`, then the input is read from stdin.
```

Comment:
I tried to keep the same standard as `mix format`, where the `-` filename toggles _stdin_ input (instead of having a `--stdin` flag option):

https://github.com/elixir-lang/elixir/blob/main/lib/mix/lib/mix/tasks/format.ex#L11